backend/src/serverless/integrations/services/integrationProcessor.ts

```diff
@@ -39,7 +38,6 @@ export class IntegrationProcessor extends LoggerBase {
       new TwitterReachIntegrationService(),
       new SlackIntegrationService(),
       new GithubIntegrationService(),
```

Comment:
When I integrated devto and linkedin I not only just removed the services here but also removed the actual service files + external api request files/types (usecases). Just to have a nice cleanup in the backend folder as wel.